scripts/current_version.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+# Used with Bazel's stamping feature
+echo BUILD_SCM_VERSION ${BUILD_SCM_VERSION_OVERRIDE:-$(git describe --abbrev=7 --tags HEAD)}
```

Comment:
@thesayyn Based on https://git-scm.com/docs/git-describe default value of --abbrev is 7 and it's used to change the default value. This means we might remove it if there's not any special reason for you to put it there.

Reply:
It might be different in older versions of git. This is the reason why I have put it there.

Reply:
Makes sense. Thanks.